controllers/snackController.js

```diff
@@ -16,21 +21,28 @@ snacks.get("/", async (req, res) => {
 snacks.post("/", async (req, res) => {
   try {
     const snack = await createSnack(req.body);
-    res.json(snack);
+    res.status(200).json(snack);
   } catch (error) {
     res.status(400).json({ error });
   }
 });
 
+// UPDATE
+snacks.put("/:id", async (req, res) => {
+  const { id } = req.params;
+  const snack = await updateSnack(id, req.body);
+  res.status(200).json(snack);
+});
```

Comment:
Looks good! Random question, should we add error handling in case snack is not returned due to error?

Reply:
Good call! I'll add that now

Reply:
Looks good!